components/renku_data_services/session/api.spec.yaml

```diff
@@ -514,9 +623,10 @@ components:
       description: Kind of environment to use
       type: string
       enum:
-        - GLOBAL
-        - CUSTOM
-      example: CUSTOM
+        - global
+        - custom
+        - builder
```

Comment:
Changing the casing creates an API breaking change. Do we want this?

Reply:
See comments above, currently our API is inconsistent in terms of casing (most enums are lower case, this file is one of the few places where it's upper case). This isn't really related to this PR, my thinking was just, if we change the API here anyways, might be a good point to make it consistent

Reply:
I undid this change since it requires a database migration as well. I'll re-introduce it later once working on the db.